exercises/practice/forth/.meta/config.json

```diff
@@ -16,7 +17,5 @@
       ".meta/lib/example.dart"
     ]
   },
-  "blurb": "Implement an evaluator for a very simple subset of Forth.",
-  "source": "Exercism JS track",
```

Comment:
Isn't there some way to mention when an exercise stems from another track?

Reply:
There is! I totally missed that this got deleted. I will replace it with the forked from entry that we settled on.

Reply:
I was wrong. We don't have this for practice exercises, as they're assumed to be implemented based on problem-specifications, or made up from whole cloth.

We may consider introducing authors and contributors in problem-specifications as well, but we haven't done so yet.

Reply:
Ok, that may have been what I was thinking of. Thanks